Airwallex/Core/Sources/Internal/Extensions/NSObject+Utils.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  NSObject+Utils.swift
+//  Core
+//
+//  Created by Tony He (CTR) on 2024/7/24.
+//  Copyright © 2024 Airwallex. All rights reserved.
+//
+
+import Foundation
+import os
+
+@objc
+public extension NSObject {
```

Comment:
Does it serve same purpose as "NSObject+Logging"? Do we need to implement it again?

Reply:
swift does not support varargs, in future I think we can use the swift one mostly. currently, I feel that adding #ifdef everywhere to use this swift func is not very worthwhile.

Reply:
Sure, we can use #ifdef only in NSObject+Logging, so from where we create `formattedMessage` we utilise `[self logMessage:formattedMessage]` to reuse same logic. And then NSObject+Logging becomes a shell/wrapper for OC invocation.